### ISSUE TITLE
telescope command shows wrong memory dump if the content is zero

### ISSUE THREAD

**Issue by sferrini** [hugsy/gef#197]
### Step 1: Describe your environment

  * Operating System: GNU/Linux ubuntu-xenial 4.4.0-101-generic
  * Architecture: x86_64
  * GDB version: GNU gdb (Ubuntu 7.11.1-0ubuntu1~16.5) 7.11.1
  * Python library version: libpython3.5m.so.1.0

#### Steps to reproduce/Observed Results

```
gef➤  p calloc(128, 1)
$2 = (void *) 0x6024e0

gef➤  telescope 0x6024e0
0x00000000006024e0│+0x00: 0x00000000006024e0
0x00000000006024e8│+0x08: 0x00000000006024e8
0x00000000006024f0│+0x10: 0x00000000006024f0
0x00000000006024f8│+0x18: 0x00000000006024f8
0x0000000000602500│+0x20: 0x0000000000602500
0x0000000000602508│+0x28: 0x0000000000602508
0x0000000000602510│+0x30: 0x0000000000602510
0x0000000000602518│+0x38: 0x0000000000602518
0x0000000000602520│+0x40: 0x0000000000602520
0x0000000000602528│+0x48: 0x0000000000602528

gef➤  x/10g 0x6024e0
0x6024e0:	0x0	0x0
0x6024f0:	0x0	0x0
0x602500:	0x0	0x0
0x602510:	0x0	0x0
0x602520:	0x0	0x0
```
#### Expected results

```
gef➤  telescope 0x6024e0
0x00000000006024e0│+0x00: 0x0000000000000000
0x00000000006024e8│+0x08: 0x0000000000000000
0x00000000006024f0│+0x10: 0x0000000000000000
0x00000000006024f8│+0x18: 0x0000000000000000
0x0000000000602500│+0x20: 0x0000000000000000
0x0000000000602508│+0x28: 0x0000000000000000
0x0000000000602510│+0x30: 0x0000000000000000
0x0000000000602518│+0x38: 0x0000000000000000
0x0000000000602520│+0x40: 0x0000000000000000
0x0000000000602528│+0x48: 0x0000000000000000
```

**Comment by hugsy** [comment]
Hi,

Thanks for reporting this regression. I'll fix asap.

**Comment by hugsy** [comment]
Fixed by commit 92e5302

```
gef➤  p calloc(128, 1)
$1 = (void *) 0x620010
gef➤  dps $1
0x0000000000620010│+0x00: 0x0000000000620010
0x0000000000620018│+0x08: 0x0000000000000000
0x0000000000620020│+0x10: 0x0000000000000000
0x0000000000620028│+0x18: 0x0000000000000000
0x0000000000620030│+0x20: 0x0000000000000000
0x0000000000620038│+0x28: 0x0000000000000000
0x0000000000620040│+0x30: 0x0000000000000000
0x0000000000620048│+0x38: 0x0000000000000000
0x0000000000620050│+0x40: 0x0000000000000000
0x0000000000620058│+0x48: 0x0000000000000000
gef➤  x/10gx $1
0x620010:       0x00000000fbad240c      0x0000000000000000
0x620020:       0x0000000000000000      0x0000000000000000
0x620030:       0x0000000000000000      0x0000000000000000
0x620040:       0x0000000000000000      0x0000000000000000
0x620050:       0x0000000000000000      0x0000000000000000
```

Thank you!

**Comment by sferrini** [comment]
I can confirm that now it's fixed, thanks for being so fast!
